index.html

```diff
@@ -1,3 +1,46 @@
-﻿
-
+<html lang ="en">
+<img src="coderbabez-hw-01-html-basics-megknoll/img/cookies.jpg" alt="Cookies">
+<<head>The Best Chocolate Chip Cookies By Grandma
```

Comment:
Be careful with your brackets! Opening tags should just be `<tag>` and closing is `</tag>`. Take a look at where you might have too many opening brackets